### PR TITLE
Initial team compass docs

### PR DESCRIPTION
**Pull request by Zsailer**
Borrowed from the [Jupyter Server Team Compass repo.](https://github.com/jupyter-server/team-compass)

This PR doesn't name any team members. That can happen in a separate PR.

### CODE REVIEW

**Comment by Zsailer** [comment]
Pinging @tonyfast, @jtpio, and @afshin for review.

**Review by jtpio — APPROVED** [review]
Looks good, thanks!

File: docs/_build/html/_static/jupyter_server_logo.svg [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (jtpio) [inline docs/_build/html/_static/jupyter_server_logo.svg:1]: Looks like we could do a quick round of cleanup in follow-up PRs, for example this file seem to be copied from the Jupyter Server team compass.

**Review by willingc — COMMENTED** [review]
@Zsailer Nice to see this. Did you mean to include the `_build` directory? Usually that is gitignored.

**Comment by Zsailer** [comment]
Oops! Thanks @willingc and @jtpio. I didn't mean to checkin the `_build` dir. It looks like I lost the `.gitignore` at some point, which would have ignored that directory. I'll amend the last commit to drop that directory from the history.

File: docs/_build/html/_static/jupyter_server_logo.svg (hunk 1 above)
Comment (Zsailer) [reply docs/_build/html/_static/jupyter_server_logo.svg]: Deleted this directory—this was supposed to be ignored. Thanks!

**Review by willingc — APPROVED** [review]

**Comment by damianavila** [comment]
Looks nice,  @Zsailer! Thanks for pushing on this one!

**Comment by Zsailer** [comment]
This page is now live! 

https://jupyter-notebook-team-compass.readthedocs.io/en/latest/index.html

**Pull request merged**